File: solutions/0344-reverse-string/solution.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        length = len(s)
        l, r = 0, length-1
        while l <= r:
            temp = s[l]
            s[l] = s[r]
            s[r] = temp
            l += 1
            r -= 1
        

File: solutions/0344-reverse-string/solution.java
class Solution {
    public void reverseString(char[] s) {
        int len = s.length;
        char temp;
        int l = 0;
        int r = len - 1;
        while(l <= r)
        {
            temp = s[l];
            s[l] = s[r];
            s[r] = temp;
            l++;
            r--;
        }
    }
}
